packages/front-end/components/Layout/TopNav.module.scss

```diff
@@ -11,9 +13,14 @@ $sidebarwidth: 240px;
   transition: all 0.5s cubic-bezier(0.685, 0.0473, 0.346, 1);
   background-color: var(--surface-background-color);
   border-bottom: 1px solid var(--border-color-300);
+  padding: 0 16px;
+  @media (max-width: 520px) {
```

Comment:
I noticed we're not doing this anywhere in our app but I'm wondering if we wanted to start using the breakpoints configured in the bootstrap theme, e.g. https://getbootstrap.com/docs/5.3/layout/breakpoints/#min-width

Reply:
we certainly could, maybe something to tackle across the board as part of a design review (hopefully soon!!)